Justfile

```diff
@@ -114,7 +114,7 @@ integration-tests *test:
   #!/bin/bash
   set -euo pipefail
   testName=${1:-}
-  go test -fullpath -count 1 -v -tags integration -run "$testName" ./integration ./backend/controller/cronjobs
+  go test -fullpath -count 1 -v -tags integration -run "$testName" $(git grep -l --untracked '^//go:build integration' | xargs -I {} dirname './{}' | sort | uniq)
```

Comment:
You should be able to just use `./...` here and Go will figure it out.

Reply:
excellent, should the `integration-run` target in `.github/workflows/ci.yml` also replace this grep sequence with `./...`?

Reply:
Unfortunately not, in that location the shell fragment is trying to quickly figure out what the test shards are so it should use something like the above. However in this location that's not an issue as the tests aren't being sharded.